### DISCUSSION TITLE
Rareté basique

### DISCUSSION THREAD

**Discussion by BananePlantain** [Crownicles/Crownicles#2022] [Suggestions fermées]
Salut,
je propose de supprimer la rareté "basique" décrite comme "Rareté de l'item possédé en début de jeu" puisque le joueur ne possede aucun objets en début de jeu,aucun objet n'as cette rareté,elle est indisponible et même le poing (quand on a pas d'arme) n'as de rareté affiché.
BananePlantain

**Comment by romain22222** [comment]
nooooooooooooooooooooooooooooooooooooooooooooooooo comment OSES TU VOLER LE NUMERO 2022 !

**Reply by BananePlantain** [reply]
^^

**Comment by symsym-1629** [comment]
on peut pas la supprimer au sens propre, un item est obligé d'avoir une rareté et ca vaut pour les poings

**Reply by BananePlantain** [reply]
![poing](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/126412670/5b1b73c2-b555-4bb0-b67c-ec2aeb78e7c4)
👀

**Comment by Nargacaura** [comment]
Et par quoi tu veux remplacer la rareté basique sans faire crash le bot en checkant l'inventaire d'un joueur sans équipement de tel type ou ne possédant que le poing comme arme? Faut bien qu'il y ait une rareté 0 pour contrebalancer les objets trouvés qui ont une rareté entre 1 et 8 (entre commun et mythique).

**Reply by BananePlantain** [reply]
effectivement oublie ca
